### ISSUE TITLE
Display images directly on terminal

### ISSUE THREAD

**Issue by fowlslegs** [pazz/alot#775]
It would be nice to be able to view image attachments directly in the terminal. It seems there are a few Python libraries for doing this. I can't say which one works best.

**Comment by josch** [comment]
You mean.... via ASCII art?

**Comment by fowlslegs** [comment]
No. Check out the demo on https://github.com/DTVD/rainbowstream. Basically they can be displayed in really shoddy resolution. Good enough to get an idea of what the images are, but doesn't expose the user to the security risk of opening an arbitrary image file.

**Comment by MacGyverNL** [comment]
Well you're still opening arbitrary image files. Just with python, with a script designed to parse them into a terminal dump. That might still have security vulnerabilities, but, granted, probably less so than the average everything-and-the-kitchen-sink-image-library.

**Comment by fowlslegs** [comment]
You could actually just do this yourself with .mailcap, no?

**Comment by geier** [comment]
If you're using xterm or urxvt you can use w3img, like [ranger](http://ranger.nongnu.org/) is doing it.

**Comment by varac** [comment]
Just ran across [timg - A terminal image viewer](https://github.com/hzeller/timg).
